junction/profiles/views.py

```diff
@@ -0,0 +1,17 @@
+from django.shortcuts import render
+
+from junction.proposals.models import Proposal
+from django.contrib.auth.decorators import login_required
+from django.views.decorators.http import require_http_methods
+
+
+@login_required
+@require_http_methods(['GET'])
+def dashboard(request):
+    proposal_qs = Proposal.objects.order_by('conference__end_date')
+    user_proposal_list = []
+    if request.user.is_authenticated():
```

Comment:
This is repetitive, decorator `@login_required` takes care of auth. You can remove this check.

We need to display proposals based on conference.
View will look like
```
conferences = Conference.objects.order_by('end_date')
return render(request, 'profiles/dashboard.html', {'conferences': conferences})
```